<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="12" passed="8">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="1845" started-at="2017-09-30T08:40:46Z" finished-at="2017-09-30T08:40:48Z">
    <groups>
      <group name="regression">
        <method signature="driver_summation.print(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="print" class="sep_dataprovider.driver_summation"/>
      </group> <!-- regression -->
      <group name="smoke">
        <method signature="driver_summation.test_summation(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="test_summation" class="sep_dataprovider.driver_summation"/>
      </group> <!-- smoke -->
    </groups>
    <test name="Default test" duration-ms="1845" started-at="2017-09-30T08:40:46Z" finished-at="2017-09-30T08:40:48Z">
      <class name="sep_dataprovider.driver_summation">
        <test-method status="PASS" signature="startextentreports()[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="startextentreports" is-config="true" duration-ms="284" started-at="2017-09-30T14:10:46Z" finished-at="2017-09-30T14:10:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startextentreports -->
        <test-method status="PASS" signature="print(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="print" duration-ms="12" started-at="2017-09-30T14:10:47Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=3.0, Exp_Result=5.0, Num1=2.0}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- print -->
        <test-method status="PASS" signature="print(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="print" duration-ms="4" started-at="2017-09-30T14:10:47Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=4.0, Exp_Result=7.0, Num1=3.0}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- print -->
        <test-method status="PASS" signature="print(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="print" duration-ms="3" started-at="2017-09-30T14:10:47Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=5.0, Exp_Result=6.0, Num1=4.0}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- print -->
        <test-method status="PASS" signature="print(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="print" duration-ms="2" started-at="2017-09-30T14:10:47Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=6.0, Exp_Result=7.0, Num1=5.0}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- print -->
        <test-method status="PASS" signature="print(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="print" duration-ms="2" started-at="2017-09-30T14:10:47Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=7.0, Exp_Result=2.0, Num1=6.0}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- print -->
        <test-method status="PASS" signature="print(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="print" duration-ms="2" started-at="2017-09-30T14:10:47Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=8.0, Exp_Result=15.0, Num1=7.0}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- print -->
        <test-method status="PASS" signature="test_summation(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="test_summation" duration-ms="628" started-at="2017-09-30T14:10:47Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=3.0, Exp_Result=5.0, Num1=2.0}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_summation -->
        <test-method status="PASS" signature="test_summation(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="test_summation" duration-ms="77" started-at="2017-09-30T14:10:48Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=4.0, Exp_Result=7.0, Num1=3.0}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_summation -->
        <test-method status="FAIL" signature="test_summation(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="test_summation" duration-ms="100" started-at="2017-09-30T14:10:48Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=5.0, Exp_Result=6.0, Num1=4.0}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
null
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:37)
	at sep_dataprovider.driver_summation.test_summation(driver_summation.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_summation -->
        <test-method status="FAIL" signature="test_summation(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="test_summation" duration-ms="102" started-at="2017-09-30T14:10:48Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=6.0, Exp_Result=7.0, Num1=5.0}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
null, null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
null, null
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:37)
	at sep_dataprovider.driver_summation.test_summation(driver_summation.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_summation -->
        <test-method status="FAIL" signature="test_summation(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="test_summation" duration-ms="70" started-at="2017-09-30T14:10:48Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=7.0, Exp_Result=2.0, Num1=6.0}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
null, null, null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
null, null, null
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:37)
	at sep_dataprovider.driver_summation.test_summation(driver_summation.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_summation -->
        <test-method status="FAIL" signature="test_summation(java.util.Map)[pri:0, instance:sep_dataprovider.driver_summation@73ad2d6]" name="test_summation" duration-ms="65" started-at="2017-09-30T14:10:48Z" data-provider="dp_sum" finished-at="2017-09-30T14:10:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Num2=8.0, Exp_Result=15.0, Num1=7.0}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
null, null, null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
null, null, null
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:37)
	at sep_dataprovider.driver_summation.test_summation(driver_summation.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_summation -->
      </class> <!-- sep_dataprovider.driver_summation -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
